### PR TITLE
feat: add tokenized vaults to erc4626 registry

### PR DESCRIPTION
**Pull request by Xeonus**
- add gnosis and mainnet tokenized vaults as per code review
- checked vault and asset as per PROD boosted pools in [test UI](https://test.balancer.fi/pools?networks=MAINNET,GNOSIS&protocolVersion=3)
- please cross-check correct setup. Goal is for tokenized vault warnings to go away

IMPORTANT: main registry with ERC4626 rate providers still needs to be populated to register boosted pool rate providers as well!

Example: [Boosetd pool with waGNO and wstETH tokenized vaults](https://test.balancer.fi/pools/gnosis/v3/0x272d6be442e30d7c87390edeb9b96f1e84cecd8d)
image

### CODE REVIEW

**Comment by Xeonus** [comment]
Note: need to wait on final review and also merge on rate providers like https://github.com/balancer/code-review/pull/200

**Review by mkflow27 — COMMENTED** [review]
Added the review information. Waiting for Daniel to approve

**Review by mkflow27 — APPROVED** [review]

**Review by danielmkm — APPROVED** [review]
Great work getting these done

**Pull request merged**
